### PR TITLE
UI: Added cypress test for Data quality and profiler

### LINKED ISSUE #7420: Cypress test for data-quality and profiler
Cover Data-quality and profiler feature in cypress test
Points need to be covered:-
- Profiler
- [x] Add mysql service
- [x] Ingest mysql data
- [x] Ingest profiler
- [x] Check if profiler data is present or not

- Data Quality
- [x] Create new test suite
- [x] Create table test case
- [x] Edit table test case
- [x] Delete table test case
- [x] Create Column test case
- [x] Edit Column test case
- [x] Delete Column test case
- [x] Delete test suite

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I worked on the adding [Cypress test for data-quality and profiler#7420](https://github.com/open-metadata/OpenMetadata/issues/7420)
Closes #7420

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js [hunk 1]
```diff
@@ -52,25 +58,45 @@ export const handleIngestionRetry = (type, testIngestionButton, count = 0) => {
...
-        cy.get('.tableBody-row > :nth-child(4)').should('have.text', 'Success');
+    cy.get(`.tableBody > :nth-child(${rowIndex}) > :nth-child(4)`).then(
+      ($ingestionStatus) => {
+        if (
+          ($ingestionStatus.text() === 'Running' ||
+            $ingestionStatus.text() === 'Queued') &&
+          retryCount <= retryTimes
+        ) {
+          // retry after waiting for 20 seconds
+          cy.wait(20000);
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/common/common.js:69]: Can we check if we can use waitFor here instead hardcoded timer?

**Review by chirag-madlani — APPROVED** [review]
@ShaileshParmar11 A minor comment rest looks good

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/common/common.js:69]: Hello, This is to wait for run ingestion on the airflow side, as we don't make any API call from UI, it is done from the ingestion side, and we only get status once its a success, so we have to hardcoded the timer!

**Comment by Sachin-chaurasiya** [comment]
@ShaileshParmar11 , Some cypress tests are failing, can you check?

**Comment by ShaileshParmar11** [comment]
> @ShaileshParmar11 , Some cypress tests are failing, can you check?

Looking into it!

**Pull request merged**